components/TARGET_PSA/TARGET_TFM/COMPONENT_NSPE/interface/src/tfm_psa_ns_api.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2018, Arm Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
this new component will need a license files: https://os.mbed.com/docs/mbed-os/v5.11/contributing/license.html (follow using different license section please)

Reply:
By the time this is ready, there will be list of non apache components in Readme.md, this will need to be added tehre as well.

Reply:
My understanding is that this can't be release under Apache 2.0?

Reply:
CC @amiraloosh 

There is a ticket opened: anaqua record ID 81133658 which should handle this issue.

Also these are files which are imported in PR #9653, so I guess that PR should also hold for this approval, right?